Essentials/src/main/java/com/earth2me/essentials/utils/LocationUtil.java

```diff
@@ -266,6 +266,7 @@ public static boolean shouldFly(IEssentials ess, final Location loc) {
         int y = (int) Math.round(loc.getY());
         final int z = loc.getBlockZ();
         int count = 0;
+        //Check whether more than 2 unsafe block are below player.
```

Comment:
Please follow the existing conventions for code comments (space between // and comment).

Reply:
Yes of course, I'll read them next time I fix a bug, I wrote that quickly and I should have probably take more time for all theses details